cra/task-bom.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     - name: revision
       description: the revision
       default: master
+    - name: pr-repository
```

Comment:
Considering that we can run task in CI workflow as well. Can we just use `repository` as the variable name to make it generic ? We can pass the right value to the task from pipeline.

Reply:
The cra-bom cra-cis-check and cis-vulnerability-scan  need both the target repo and the source repo (in case of pr pipeline). We cannot combine them.

I change the **pr-repository** to **source-repository** in the task. I have not changed the **repository** to **target-repository** as this is a bigger change. I will create a separate work item to make that change.